lldap_cli_wrapper.py

```
@@ -67,34 +77,65 @@ def create_user(uid, email, displayname=None, firstname=None, lastname=None):
+                },
+                timeout=10,
+            )
+            if response.status_code not in {200, 204}:
+                return (
+                    False,
+                    f"User created but Authelia reset failed: {response.status_code} - {response.text}",
+                )
+
+        except requests.RequestException as e:
+            return False, f"User created but Authelia reset failed (network error): {e!s}"
+        except Exception as e:  # noqa: BLE001
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)